src/de.rs

```diff
@@ -33,6 +33,37 @@ pub fn from_slice<'de, T>(bytes: &'de [u8]) -> Result<T, Error>
 ///
 /// This function will attempt to interpret `s` as a TOML document and
 /// deserialize `T` from the document.
+///
+/// # Examples
+///
+/// ```no_run
```

Comment:
The reviewer should have explained it better but in that case it was because aborting the test process counts as failing the test. Understandably the test harness does not treat an aborted test as successful. In that case the test was an example for `std::process::abort` so things were different.

Yes examples should always be tested if feasible.